table.rst

```diff
@@ -0,0 +1,40 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              | pd.Int8Dtype,           |
+|                   | pd.Int16Dtype,          |
+|                   | pd.Int32Dtype,          |
+|                   | pd.Int54Dtype,          |
+|                   | "int",int,np.int8,      |
+|                   |  np.int16,np.int32,     |
+|                   |  np.int64,np.uint8      |
+|                   |  np.uint16,np.uint32    |
+|                   |  np.uint64,np.intp      |
+|                   |  np.uintp,              |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | np.byte,np.ubyte,bytes  |
++-------------------+-------------------------+
+|  float            | pd.Float32Dtype,        |
+|                   | pd.Float64Dtype,        |
+|                   | float,"float",np.float16|
+|                   | np.float32,np.float64   |
+|                   | np.float96,np.float128  |
++-------------------+-------------------------+
+|  complex          | "complex",np.complex64, |
+|                   | np.complex128,          |
+|                   | np.complex192,          |
+|                   | np.complex256           |
++-------------------+-------------------------+
+|  bool             | pd.BooleanDtype, bool   |
+|                   | np.bool_,               |
++-------------------+-------------------------+
+|                   | np.timedelta64          |
++-------------------+-------------------------+
+|  np.datetime64    |                         |
```

Comment:
above two rows should have `np.datetime64` and `np.timedelta64` on the left column.  Right column should include those types plus the corresponding `datetime` types.